services/efs-guardian/tests/unit/conftest.py

```diff
@@ -140,23 +129,37 @@ async def client(app: FastAPI) -> AsyncIterator[httpx.AsyncClient]:
         yield client
 
 
+#
+# Redis
+#
+
+
 @pytest.fixture
-async def rpc_client(
-    rabbit_service: RabbitSettings,
-    app: FastAPI,
-    rabbitmq_rpc_client: Callable[[str], Awaitable[RabbitMQRPCClient]],
-) -> RabbitMQRPCClient:
-    return await rabbitmq_rpc_client("client")
+def disable_redis_and_background_tasks_setup(mocker: MockerFixture) -> Callable:
```

Comment:
Q: does it even make sense that you can disable REdis? Ah I see that is only for tests.
BTW, I use extensively in many places fakeredis, which is in memory redis. Which you are removing. Maybe I know why?

Reply:
Yes, if you check all the tests in the beginning use `app_environment` with additional options: 
- with_disabled_redis_and_background_tasks
- with_disabled_postgres
- with_disabled_redis_and_background_tasks
The idea is that in the beginning of the test is nicely readable what to expect. For now, it is sufficient for me, if I will use Redis more yes I can add `fakeredis`. For now, I do not need it.

Reply:
I am not sure I follow as you had fakeredis until this PR. but ok, as you wish.

Reply:
No no, my point is that I am not using the "real" or "fake" redis anywhere in the tests. All tests currently have redis disabled.